src/test/cls_rbd/test_cls_rbd.cc

```diff
@@ -1797,6 +1798,7 @@ TEST_F(TestClsRbd, mirror_image) {
 }
 
 TEST_F(TestClsRbd, mirror_image_status) {
+  SKIP_IF_CRIMSON();
```

Comment:
It would be nice if
- the commit title reflected that it's only skipped in Crimson, e.g. "test/cls_rbd/test_cls_rbd: Skip mirror_image_status (Crimson)"
- the commit message explained the reason, e.g. "XYZ is currently not supported"

Reply:
> This is an attempt to establish a Crimson baseline and from there we will resolve the skipped tests.

"Establishing a baseline" is a perfectly OK thing to put in the commit message ;)

Reply:
Updated.